src/ComplexTypes/GetSignatureRequest.php

```diff
@@ -14,23 +14,23 @@ class GetSignatureRequest extends BaseType
     protected $Customer = null;
 
     /**
-     * @var RequestSignature $Shipment
+     * @var RequestShipment $Shipment
```

Comment:
Yes, you're right. There's a bug, now that I look at it I see that it stems from CIF's re-usage of `Shipment` data that contains less info than the `Shipment` for other requests, so they had to give the complextype a different name.

Still doesn't belong in this PR though. You're welcome to create an issue or another PR. If not, I'll fix it if I a have bit more time later today.